src/main/java/it/pn/frontend/e2e/pages/mittente/PiattaformaNotifichePage.java

```diff
@@ -2083,4 +2083,23 @@ public void verificaBannerAttivoEInserimentoManualeSelezionato() {
         logger.info("Messaggio di alert correttamente visualizzato e coerente con quello atteso");
 
     }
+
+    public boolean attesaNotificaConIUN(String iun) {
+        int maximumRetry;
+        boolean cellaPresente = false;
+        for (maximumRetry = 0; maximumRetry <= 8; maximumRetry++) {
+            List<WebElement> cellaCodiceIUN = driver.findElements(By.xpath("//table[@id='notifications-table']//tr//td[contains(text(), '" + iun + "')]"));
+            if (!cellaCodiceIUN.isEmpty()) cellaPresente = true;
+            if (cellaPresente) {
+                getNotificationSingletonParam().setScenarioIun(hooksNew.getScenario(), iun);
+                return true;
+            } else {
+                webTool.waitTime(90);
+                maximumRetry++;
+                logger.info("Tentativo n.{}", maximumRetry);
+                driver.navigate().refresh();
+            }
+        }
+        return false;
```

Comment:
Cosa ne pensi di rendere ll metodo seguente un po' più chiaro come suggerito sotto?
```suggestion
    public boolean attesaNotificaConIUN(String iun) {
        int maxTentativi = 8;
        for (int tentativo = 0; tentativo <= maxTentativi; tentativo++) {
            List<WebElement> cellaCodiceIUN = driver.findElements(By.xpath("//table[@id='notifications-table']//tr//td[contains(text(), '" + iun + "')]"));
            if (!cellaCodiceIUN.isEmpty()) {
                getNotificationSingletonParam().setScenarioIun(hooksNew.getScenario(), iun);
                return true;
            }
            logger.info("Tentativo n.{}: notifica non trovata!", tentativo); 
            webTool.waitTime(90);            
            driver.navigate().refresh();
        }
        return false;
```

Reply:
Concordo, è più compatto e si legge meglio così, grazie